.github/scripts/setup-manager-self-hosted.py

```diff
@@ -21,13 +36,6 @@ def initialize_manager_hosted():
 
     # Catch any exception that occurs so that we can gracefully teardown
     try:
-        # wait until machine launch is complete
-        with cd(manager_home_dir):
-            with settings(warn_only=True):
-                rc = run("timeout 20m grep -q '.*machine launch script complete.*' <(tail -f machine-launchstatus)").return_code
-                if rc != 0:
-                    run("cat machine-launchstatus.log")
-                    raise Exception("machine-launch-script.sh failed to run")
 
```

Comment:
```suggestion
    with warn_only():
        run("cat /machine-launchstatus.log")
        run("env | sort")
        run("conda info")
```

I'm going to add some temporary stuff for debugging before asking to be able to ssh to the CI hosts.